src/lob-address-elements.js

```diff
@@ -152,7 +110,7 @@ import countryCodes from './country-codes.js';
      * @param {*} state - page state
      */
     function _LobAddressElements($, cfg, state) {
-      var config = {
+      let config = {
```

Comment:
This is great! Thx for making updates to variable and function declarations. It's nice to have the updated syntax.

Reply:
Agreed! After our main work I'd like to refactor lob-address-elements to use modern syntax including javascript classes.